### PR TITLE
e2e: enable stickers and update release notes

### PR DESCRIPTION
**Pull request by churik**
* enabled unblocked test
* removed from release notes communities and regression issues, added some missing items

### CODE REVIEW

**Review by pavloburykh — APPROVED** [review]

**Comment by qoqobolo** [comment]
@churik should we also add https://github.com/status-im/status-mobile/pull/13776 to `Improvements`?

File: test/appium/tests/critical/chats/test_1_1_public_chats.py [hunk 1]
```diff
@@ -1118,7 +1118,6 @@ def test_ens_mention_push_highlighted_public_chat(self):
         self.errors.verify_no_errors()
 
     @marks.testrail_id(702157)
-    @marks.xfail(reason='Failing due to the known IPFS issue', run=False)
     def test_sticker_1_1_public_chat_mainnet(self):
         self.home_2.status_in_background_button.click_if_shown()
         [home.home_button.double_click() for home in (self.home_1, self.home_2)]
```
Comment (qoqobolo) [inline test/appium/tests/critical/chats/test_1_1_public_chats.py:1123]: Should we also add https://github.com/status-im/status-mobile/pull/13776 to Improvements?

**Comment by churik** [comment]
@qoqobolo in general should be a part of Mutual contact request support, so don't think worth separate mention

File: test/appium/tests/critical/chats/test_1_1_public_chats.py (hunk 1 above)
Comment (churik) [reply test/appium/tests/critical/chats/test_1_1_public_chats.py:1123]: @qoqobolo in general should be a part of Mutual contact request support, so don't think worth separate mention

**Review by qoqobolo — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**